Watershed Algorithm added

Ayush12062000 opened the pull request:
- Readme for watershed algorithm updated.
 -  input and resultant(output) images added
 -  Jupyter notebook for the code added. (if required can add .py file as well?)

kindly review and merge the PR.

FOR ISSUE #73

Ayush12062000 committed:
Watershed Algorithm added

geekquad reviewed (approved):
Hey @Ayush12062000, amazing work.
Just a few amendments:
* I can see a lot of unnecessary libraries. Please remove them.
* Please add a sample input and output image in the notebook itself.

Thankyou.

Ayush12062000 committed:
Watershed Algorithm Notebook(made necessary changes)

Ayush12062000 commented:
1.  I removed one library that I wasn't using in the code. Rest each and every Library has been used in the code.
2.  Added sample input and output images in the notebook.

Have a look. @geekquad

geekquad reviewed (approved):
Hey @Ayush12062000, amazing work.
